docs/tutorials/07-batch-infer.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>Before analyzing the results in the visual dashboards, run another batch infer job, but this time with a lower drift threshold, to get a drifted result. The drift decision rule is the value per-feature mean of the <a href="../monitoring/model-monitoring-deployment.html#common-terminology" target="_blank">Total Variance Distance (TVD) and Hellinger distance</a> scores. By default, the threshold is 0.7 but you can modify it through the batch infer process. As seen above, the drift result in this case was ~0.3. Reduce the threshold value to 0.2 in the following run to generate the drifted result:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>